src/types.ts

```diff
@@ -3,8 +3,6 @@ import { AxiosBasicCredentials, AxiosProxyConfig, AxiosInstance } from 'axios';
 import { DocumentNode } from 'graphql';
 import express from 'express';
 
-export default Router;
```

Comment:
We no longer need to export `Router` here, since nothing is importing it from this module anymore. Prior to these changes, TypeScript applications would import the default from here, and JavaScript applications would import the default from index